book/src/become-a-validator-source.md

```diff
@@ -0,0 +1,130 @@
+# Become an Validator: Building from Source
+
+## 1. Download and install Lighthouse
+
+If you already have Rust installed, you can install Lighthouse with the
+following three commands:
+
+- `$ git clone https://github.com/sigp/lighthouse.git`
+- `$ cd lighthouse`
+- `$ make`
+
+You've completed this step when you can run `$ lighthouse --help` and see the
+help menu.
+
+> - If you're not familiar with Rust or you'd like more detailed instructions,
+>   see the [Installation Guide](./installation.md) which contains a
+>   [Troubleshooting](installation.html#troubleshooting) section.
+
+## 2. Start an Eth1 client
+
+As Eth2 relies upon the Eth1 chain for validator on-boarding and eventually
+Eth1 may use the Eth2 chain as a finality gadget. As such, all Eth2 validators
```

Comment:
I believe there's a wording error here? "As [...].  As such [..]"?
Maybe I'm completely wrong here, but I think remove ". As such, " should suffice.
"As Eth2 relies upon the Eth1 chain for validator on-boarding and eventually
Eth1 may use the Eth2 chain as a finality gadget, all Eth2 validators ..."

Reply:
Thank you!